--- git status ---
On branch bd_1
Your branch is up to date with 'origin/bd_1'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/isaaclab_tasks/isaaclab_tasks/direct/bd_1/bd_1_walkingfixed.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/isaaclab_tasks/isaaclab_tasks/direct/bd_1/bd_1_walkingfixed.py b/source/isaaclab_tasks/isaaclab_tasks/direct/bd_1/bd_1_walkingfixed.py
index 520b516a..554ef5d4 100644
--- a/source/isaaclab_tasks/isaaclab_tasks/direct/bd_1/bd_1_walkingfixed.py
+++ b/source/isaaclab_tasks/isaaclab_tasks/direct/bd_1/bd_1_walkingfixed.py
@@ -245,6 +245,7 @@ class BD1WalkingFixedEnv(DirectRLEnv):
         joint_torques = torch.sum(torch.square(self._robot.data.applied_torque), dim=1)
         # joint acceleration
         joint_accel = torch.sum(torch.square(self._robot.data.joint_acc), dim=1)
+        joint_accel[joint_accel > 25.0] = 25.0
         # action rate
         action_rate = torch.sum(torch.square(self._actions - self._previous_actions), dim=1)
         action_rate_reward = action_rate * self.cfg.action_rate_reward_scale * self.step_dt